File: src/test/java/com/rafaelfiume/tictactoe/PlayerTest.java
package com.rafaelfiume.tictactoe;

import org.junit.Test;

import static org.hamcrest.Matchers.is;
import static org.junit.Assert.assertThat;

public class PlayerTest {

    @Test
    public void playersReturnTheSymbolThatIdentifiesThem() {
        assertThat(Player.X.symbol(), is('X'));
        assertThat(Player.O.symbol(), is('O'));
    }

}

File: src/test/java/com/rafaelfiume/tictactoe/PlayerTest.kt
package com.rafaelfiume.tictactoe

import org.hamcrest.Matchers.`is`
import org.junit.Assert.assertThat
import org.junit.Test

class PlayerTest {

    @Test
    fun playersReturnTheSymbolThatIdentifiesThem() {
        assertThat(Player.X.symbol(), `is`('X'))
        assertThat(Player.O.symbol(), `is`('O'))
    }

}
